apps/financial-aid/backend/src/app/modules/municipality/municipality.controller.ts

```diff
@@ -39,19 +35,3 @@ export class MunicipalityController {
     return municipality
   }
 }
-
-// @UseGuards(JwtAuthGuard)
```

Comment:
remove?

Reply:
Yes removed this guard since we now use `IdsUserGuard` instead which is used for this whole controller 😄 

And this was just commented out code, decided to throw it in the bin 🗑️  and when we'll implement it then we will just add it back and use it 👼